Redactor fix and little clean

samuel-queniart opened the pull request:
Fix redactor error
Remove useless file
Set stable version to dependence

samuel-queniart committed:
Fix version symfony2 required for >=2.3,<3.0

samuel-queniart committed:
fixed untracked files and resolve redactor error

vincentchalamon commented on .gitignore:
```diff
@@ -6,3 +6,6 @@ Tests/App/cache/*
 Tests/App/web/*
 !Tests/App/web/uploads/.gitkeep
+
+.DS_Store
```
Should not be present in this file. You must declare common ignored files in your [global gitignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)

vincentchalamon commented on .gitignore:
```diff
@@ -6,3 +6,6 @@ Tests/App/cache/*
 !Tests/App/web/uploads/.gitkeep
+
+.DS_Store
+.idea
```
Same

vincentchalamon commented:
Please remove/ignore `.DS_Store` file

vincentchalamon commented on Resources/public/redactor/langs/de.js:
```diff
@@ -1,79 +1,79 @@
+	header1: 'Überschrift 1',
+	header2: 'Überschrift 2',
+	header3: 'Überschrift 3',
+    header4: 'Überschrift 4',
```
Wrong indent

vincentchalamon commented on Resources/public/redactor/langs/de.js:
```diff
@@ -1,79 +1,79 @@
+	header2: 'Überschrift 2',
+	header3: 'Überschrift 3',
+    header4: 'Überschrift 4',
+    header5: 'Überschrift 5',
```
Wrong indent

vincentchalamon commented on Resources/public/redactor/langs/ua.js:
```diff
@@ -1,76 +1,76 @@
+	horizontalrule: 'Горизонтальная лінійка',
+	fullscreen: 'На весь екран',
+	deleted: 'Закреслений',
+    anchor: 'Anchor',
```
Wrong indent

vincentchalamon commented on Resources/public/redactor/langs/by.js:
```diff
@@ -1,76 +1,76 @@
+    fullscreen: 'Ва ўвесь экран',
+    deleted: 'Закрэслены',
+    anchor: 'Anchor',
+	link_new_tab: 'Open link in new tab',
```
Wrong indent (same for following lines)

vincentchalamon commented on composer.json:
```diff
@@ -15,7 +15,7 @@
         "sensio/framework-extra-bundle": "3.0.*",
         "symfony/symfony": ">=2.3,<3.0",
-        "gregwar/form-bundle": "@stable"
+        "gregwar/form-bundle": "dev-master"
```
`~2.7` instead ? Why do you need `dev-master` version ?

samuel-queniart committed:
Clean for PR
Move tab to 4 space
Remove .DS_Store and .idea to .gitignore after remove to project
Fix bundle version to 2.7 version

vincentchalamon commented:
:+1:
